### ISSUE TITLE
`MaterialDialog.datepicker` not acessible

### ISSUE THREAD

**Issue by strangesource** [vanpra/compose-material-dialogs#96]
With version 0.5.0, `MaterialDialog.datepicker` is no longer available. I tried the following imports:

```Kotlin
import com.vanpra.composematerialdialogs.datetime.date.datepicker
```
as well as 

```Kotlin
import com.vanpra.composematerialdialogs.datetime.datepicker.datepicker
```
and 

```Kotlin
import com.vanpra.composematerialdialogs.datetime.date.*
```

am I missing something? 🤔 

When I downgrade to `0.4.2`, the import 
```Kotlin
import com.vanpra.composematerialdialogs.datetime.datepicker.datepicker
```
works.

**Comment by wakaztahir** [comment]
Same issue !

**Comment by Tgo1014** [comment]
For me, all imports are failing. I'm reverting to the previous version for now.

![image](https://user-images.githubusercontent.com/8679058/127459560-50034f9f-5f95-49de-86f8-4215bc4f59f2.png)

**Comment by Tgo1014** [comment]
I just realized I can't revert because the old version breaks with compose 1.0.0, so I'm kinda stuck right now on what do to

**Comment by jiangdashao** [comment]
> For me, all imports are failing. I'm reverting to the previous version for now.
> 
> ![image](https://user-images.githubusercontent.com/8679058/127459560-50034f9f-5f95-49de-86f8-4215bc4f59f2.png)

Have same issue

**Comment by EugeneTheDev** [comment]
@Tgo1014 for me version`0.4.3` works fine with compose `1.0.0`

**Comment by vanpra** [comment]
Thanks for letting me know about this issue. I'm not exactly sure what's wrong but it seems like its something to do with resolving extension functions as all the classes are being resolved correctly. I'll look into it further and try and get it sorted as soon as possible.

**Comment by vanpra** [comment]
Looks like it was just an issue with the packagingOptions excludes. This should be fixed in 0.5.1 and the compose versions has also been updated.

**Comment by wakaztahir** [comment]
I can't still import datetimepicker , I can import date picker but datetimepicker is missing !!
